docs/configuration/index.md

```diff
@@ -1790,12 +1790,13 @@ This prioritization strategy lowers the priority of queries that cross any of a
 
 This strategy can be enabled by setting `druid.query.scheduler.prioritization.strategy` to `threshold`.
 
-|Property|Description|Default|
-|--------|-----------|-------|
-|`druid.query.scheduler.prioritization.periodThreshold`|ISO duration threshold for how old data can be queried before automatically adjusting query priority.|none|
-|`druid.query.scheduler.prioritization.durationThreshold`|ISO duration threshold for maximum duration a queries interval can span before the priority is automatically adjusted.|none|
-|`druid.query.scheduler.prioritization.segmentCountThreshold`|Number threshold for maximum number of segments that can take part in a query before its priority is automatically adjusted.|none|
-|`druid.query.scheduler.prioritization.adjustment`|Amount to reduce the priority of queries which cross any threshold.|none|
+|Property| Description                                                                                                                  |Default|
+|--------|------------------------------------------------------------------------------------------------------------------------------|-------|
```

Comment:
nit: please disable the auto table formatting, i don't think we use it in most places and it isn't really needed for the website rendering

Reply:
Thanks for the review @clintropolis I'm thinking it would be better as a separate option. It may cause confusion if someone specifies a large time range in their query and it does not get de-prioritized. And I can imagine some operators may want to just penalize any query with a large time range regardless of whether data is present

Reply:
Could you elaborate on disabling the auto table formatting? I'm not sure how to do that

Reply:
ah , i guess i was assuming you were using some kind of tool that added all of the extra formatting to the markdown table that caused all of these changes when you added the new entry (I believe intellij has such an option enabled by default). Anyway, I was just suggesting to minimize the lines changed to only the entry for the new option, and leave out the padding and extra `---` on the markdown table

Reply:
Ahh thanks for explaining. I think intellij did that, but I just pushed a change to fix the formatting!

Reply:
fwiw, the `FORMAT_TABLES` setting  controls markdown auto-formatting. It was disabled [here](https://github.com/apache/druid/blob/master/dev/druid_intellij_formatting.xml#L78), so [importing](https://github.com/apache/druid/blob/master/dev/intellij-setup.md#code-style) the code style file in your IDE should stop auto-formatting markdown tables